Src/Application/Score/Age/Queries/AgeScoreGetAllQueryHandler.cs

```diff
@@ -1,19 +1,19 @@
 ﻿using Application.Common.Interfaces;
+using Domain.Entities;
 using Mapster;
 using MediatR;
-using Microsoft.EntityFrameworkCore;
 
 namespace Application.Score.Age.Queries;
 
 public class AgeScoreGetAllQueryHandler : IRequestHandler<AgeScoreGetAllQuery, List<AgeScoreDto>>
 {
-    private readonly IApplicationDbContext _context;
-    public AgeScoreGetAllQueryHandler(IApplicationDbContext context)
+    private readonly IRepository<AgeScore> _repository;
```

Comment:
Rename it to _ageScoreRepository for better readability, fix it in all occurences